config/default/image_tag_patch.yaml

```diff
@@ -15,3 +15,5 @@ spec:
           value: "quay.io/cryostat/jfr-datasource:2.0.0"
         - name: RELATED_IMAGE_GRAFANA
           value: "quay.io/cryostat/cryostat-grafana-dashboard:2.0.0"
+        - name: RELATED_IMAGE_REPORTS
+          value: "quay.io/cryostat/cryostat-reports:1.0.0-SNAPSHOT
```

Comment:
It looks like there's a missing end-quote here. Did `make manifests` generate this? If so, there must be a bug with it.